### PR TITLE
Add package usb.1.3.1

### PR DESCRIPTION
**Pull request by letoh**

### CODE REVIEW

File: packages/usb/usb.1.3.1/opam [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+build: [
+  ["./configure"]
+  ["./configure" "--enable-tests"] {with-test}
+  [make]
+  [make "doc"] {with-doc}
+  [make "test"] {with-test}
+]
+
+remove: [["ocamlfind" "remove" "usb"]]
```
Comment (mseri) [inline packages/usb/usb.1.3.1/opam]: This is not necessary with opam2 (and with it you can also remove the flags line)

File: packages/usb/usb.1.3.1/opam [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+authors: "Jeremie Dimino <jeremie@dimino.org>"
+maintainer: "letoh"
+license: "BSD-3-Clause"
+homepage: "https://github.com/letoh/ocaml-usb"
+bug-reports: "https://github.com/letoh/ocaml-usb/issues"
+dev-repo: "git+https://github.com/letoh/ocaml-usb.git"
+
+depends: [
+  "ocaml" {>= "4.01.0"}
+  "ocamlfind"
```
Comment (mseri) [inline packages/usb/usb.1.3.1/opam]: Do you use this at runtime or is it a build dependency?

File: packages/usb/usb.1.3.1/opam [hunk 3]
```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+name: "usb"
+version: "1.3.1"
```
Comment (mseri) [inline packages/usb/usb.1.3.1/opam]: Name and version are redundant, opam will override them with what comes from the path. Can you remove them?

Comment (letoh) [reply packages/usb/usb.1.3.1/opam]: I will remove the two lines in the next commit. Thank you!

File: packages/usb/usb.1.3.1/opam (hunk 2 above)
Comment (letoh) [reply packages/usb/usb.1.3.1/opam]: Oasis needs it, so maybe I should leave it to oasis? or just add the build filter?

**Comment by mseri** [comment]
Thanks. It looks good now, all green

**Comment by letoh** [comment]
Thank you!

**Pull request merged**